website/docs/d/cos_bucket.html.markdown

```diff
@@ -95,6 +95,51 @@ resource "ibm_cos_bucket" "smart-us-south" {
 }
 
 ```
+
+# ibm_cos_objectlock_configuration
+
+Retrive an IBM Cloud Object Storage bucket with objectlock enabled and objectlock configuration set on the bucket.It allows objectlock configuration to be updated or deleted.But it does not allow to disable objectlock as objectloock cannot be disabled once enabled on a bucket.
+
+## Example usage
+
+```terraform
+data "ibm_resource_group" "cos_group" {
+  name = "cos-resource-group"
+}
+
+data "ibm_resource_instance" "cos_instance" {
+  name              = "cos-instance"
+  resource_group_id = data.ibm_resource_group.cos_group.id
+  service           = "cloud-object-storage"
+}
+
+data "ibm_cos_bucket" "cos_bucket" {
+  bucket_name          = "a-standard-bucket"
+  resource_instance_id = data.ibm_resource_instance.cos_instance.id
+  bucket_region        = "us-south"
+  storage_class        = "Standard"
+  object_versioning {
```

Comment:
in datasource they are computed you can't take as input

Reply:
addressed this comment.